<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewUser</class>
 <widget class="QWidget" name="NewUser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>555</width>
    <height>372</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(14, 14, 14);</string>
  </property>
  <widget class="QLabel" name="l_newuser">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>10</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 24pt &quot;.SF NS Text&quot;;
color: rgb(234, 239, 238);
</string>
   </property>
   <property name="text">
    <string>New User</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>591</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_firstname">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>230</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(207, 211, 211);
border-style:outset;
border-radius:10px;
font: 14pt &quot;Arial&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Enter your First Name</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_lastname">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>80</y>
     <width>229</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(207, 211, 211);
border-style:outset;
border-radius:10px;
font: 14pt &quot;Arial&quot;;</string>
   </property>
   <property name="placeholderText">
    <string>Enter your Last Name</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_phone">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>140</y>
     <width>230</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(207, 211, 211);
border-style:outset;
border-radius:10px;
font: 14pt &quot;Arial&quot;;</string>
   </property>
   <property name="placeholderText">
    <string>Enter your Phone Number </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_email">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>140</y>
     <width>229</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(207, 211, 211);
border-style:outset;
border-radius:10px;
font: 14pt &quot;Arial&quot;;</string>
   </property>
   <property name="placeholderText">
    <string>Enter your Email Address </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_username">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>200</y>
     <width>230</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(207, 211, 211);
border-style:outset;
border-radius:10px;
font: 14pt &quot;Arial&quot;;</string>
   </property>
   <property name="placeholderText">
    <string>Enter Username</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>200</y>
     <width>231</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(207, 211, 211);
border-style:outset;
border-radius:10px;
font: 14pt &quot;Arial&quot;;</string>
   </property>
   <property name="placeholderText">
    <string>Enter Password</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_submit">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>270</y>
     <width>159</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(250, 255, 255);
background-color: rgb(73, 199, 41);
border-style:outset;
border-radius:10px;
font: 14pt &quot;Arial&quot;;</string>
   </property>
   <property name="text">
    <string>Submit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Back">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>320</y>
     <width>159</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(250, 255, 255);
background-color: rgb(73, 199, 41);
border-style:outset;
border-radius:10px;
font: 14pt &quot;Arial&quot;;</string>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
